hyp3_metadata/templates/insar/readme.md.txt.j2

```diff
@@ -0,0 +1,298 @@
+
+----------------
+## 5. Coherence Map
+
+The spectra of the two images used for InSAR must overlap well enough to generate interferometric fringes. Areas without common overlap must be filtered out prior to InSAR generation. The correlation indicates the accuracy of the phase information, or the visibility of fringes. Areas with low correlation will have noisier phase. The magnitude of the correlation is commonly referred to as coherence. The coherence map is output as a GeoTIFF with values ranging from 0 to 1; the larger the number, the higher the coherence (and the better the correlation).
+
+The coherence map is tagged with _corr.tif
+
+----------------
+## 6. Amplitude Image
+
+The amplitude GeoTIFF indicates the calibrated radiometric backscatter from the reference granule in sigma-nought power. The image is terrain corrected using a geometric correction, but not radiometrically corrected. For more informative amplitude data, order Radiometric Terrain Corrected (RTC) products for your area of interest.
+
+The amplitude image is tagged with _amp.tif
+
+----------------
+## 7. Parameter Documentation
+
+The text file included with the product lists the key processing parameters used, including the full granule names, baseline length, number of looks, output resolution, and source DEM information.
+
+The text file is not tagged; it has a .txt extension on the base filename.
+
+----------------
+## 8. Look Vector Maps
+
+Two look vector maps are output with as GeoTIFFs. The lv_theta map indicates the SAR look vector elevation angle at each map pixel, ranging from -pi/2 (down) to pi/2 (up). The lv_phi map indicates the SAR look vector orientation angle at each map pixel, ranging from 0 (east) to pi/2 (north).
+
+*These files are optional. Select the "Include Look Vectors" option in the On Demand HyP3 Processing Options to include them in the product package.*
+
+----------------
+## 9. Incidence Angle Map
+
+The local incidence angle is defined as the angle between the incident radar signal and the local surface normal. The output is a GeoTIFF giving the local incidence angle in radians for each pixel in the scene.
+
+*This file is optional. Select the "Include Inc. Angle Map" option in the On Demand HyP3 Processing Options to include it in the product package.*
+
+----------------
+## 10. DEM used to process the data
+
+This is a copy of the Digital Elevation Model (DEM) layer used for processing. This layer is tagged with _dem.tif, and is a 32-bit float GeoTIFF.
+
+The source DEM has a geoid correction applied before it is used for processing, so height values in this file will differ from the source DEM. The DEM is down-sampled to twice the pixel spacing of the desired InSAR output pixel spacing before processing. As such, an 80-m InSAR product requires the DEM to be down-sampled to a pixel spacing of 160 m. It is provided in the same pixel spacing/alignment and projected to the same UTM Zone as the interferogram files.
```

Comment:
Is the DEM GeoTIFF in the same pixel spacing as the output products? Or is it still in the resampled pixel spacing? Adjust if necessary.

Reply:
This ended up being captured as https://github.com/ASFHyP3/hyp3-gamma/issues/254

During processing we first resample the dem to twice the pixel size of the output product (160 or 80), then we resample *that* version to the pixel size of the output product (80 or 40).  What you end up with in the output product has the same pixel size as the other GeoTIFFs (80 or 40), but in-between there's possibly been some loss of resolution due to the repeated resampling.  I'm hopeful we can streamline the processing to avoid any impact on resolution.

We might eliminate the two sentences about downsampling, so we don't have to update them later if/when we decide to pursue this issue.

Reply:
@talogan Is there a reason for working with a downsampled DEM? I'd gotten the impression that it was required to be that lower resolution before being used for InSAR processing, but I don't have a sense why. 

@asjohnston-asf If we're still doing the downsampling, I think it's important to reflect that in the description. That way users don't think the DEM copy is at 80-m resolution just because the cell size is 80 m.